governance-exercises/Le Bateleur - Poll Exercise 01/Poll #1.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Adjust Spork Protocol D3M Parameters - July 24, 2023
+summary: Signal your support or opposition to adjusting the Spork Protocol DAI Direct Deposit Module (D3M) parameters.
+discussion_link: $discussion_link
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-24T16:00:00
+end_date: 2023-07-27T16:00:00
+---
+# Poll: Adjust Spork Protocol D3M Parameters - July 24, 2023
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of Spork Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 24 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+Spork Protocol is seeing significant growth in its user base and has come close to hitting its assigned Maximum Debt Ceiling. To capitalize on this momentum, the Stability Scope Facilitators and Ecosystem Actor [Alice]($link) have proposed increasing the Spork D3M Maximum Debt Ceiling, Target Available Debt and Debt Increase Cooldown parameters.
```

Comment:
Should be ceiling increase cooldown to match the docs. Other stakeholders often use other names for things. Try to stay as consistent as possible with documentation and encourage others to do the same.

Reply:
Summary is good though.